src/components/NumberInput/NumberInput.jsx

```diff
@@ -20,19 +20,19 @@ type HandleEvent = {|
       value: Object,
     },
   },
-|};
+|} | string;
```

Comment:
I'm with @kangax here too.. we should understand where this is being used with a string.